### ISSUE TITLE
MacOS job fails with: Error: An error occurred while provisioning resources (Error Type: Failure).

### ISSUE THREAD

**Issue by nullcipher** [actions/virtual-environments#1933]
**Description**  
Jobs fails with Error: 
An error occurred while provisioning resources (Error Type: Failure).
The remote provider was unable to process the request.

**Area for Triage**:  

Apple

**Question, Bug, or Feature?**:  

Bug

**Virtual environments affected**  
- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [X] macOS 10.15
- [ ] macOS 11.0
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Expected behavior**  
Jobs are expected to run smoothly.

**Actual behavior**  
Jobs fail with the error:
An error occurred while provisioning resources (Error Type: Failure).
The remote provider was unable to process the request.

**Repro steps**  
Just running a pipeline with Azure Pipelines with vmImage: macOS 10.15 or macOS-latest.

**Comment by al-cheb** [comment]
Hello, @nullcipher 
Could you please provide pipeline yaml and link to the failed build?

**Comment by tim-fitzgerald** [comment]
Seeing this as of today as well - same repro steps. Can look into providing a redacted YAML file - but this seems scoped beyond local YAML file issues and more of an Azure/Actions issue

**Comment by al-cheb** [comment]
@tim-fitzgerald, Could you please provide links to failed builds?

**Comment by tim-fitzgerald** [comment]
Is that useful if they are private within our azure org? If it is I can provide them :)

**Comment by al-cheb** [comment]
> Is that useful if they are private within our azure org? If it is I can provide them :)

Yes, please. This helps us to investigate or escalate the issue.

**Comment by nullcipher** [comment]
Hello.

We have a private org in Azure DevOps. The failed job is declared as follows:

jobs:
    - job: XcodeBuild
      pool:
        vmImage: 'macOS-10.15'

'macOS-latest' is failing also.

We are using this YAML file since May 2020 and the issue began Yesterday.

**Comment by tim-fitzgerald** [comment]
https://dev.azure.com/clio-it/autopkg/_build/results?buildId=207&view=results

@al-cheb here is a link to a failing pipeline. 

This is exact same behaviour that @nullcipher is seeing - this YAML file is unchanged for months of successful builds. This really appears to be an upstream issue with the macOS image.

**Comment by al-cheb** [comment]
@nullcipher, @tim-fitzgerald, Could you please check it should be mitigated?

**Comment by tim-fitzgerald** [comment]
@al-cheb still showing for me

**Comment by nullcipher** [comment]
@al-cheb not working.

**Comment by alepauly** [comment]
hey all, we're working on fixing the issue. You can follow here: https://status.dev.azure.com/

tl;dr - we're working on allocating more capacity to deal with the jobs coming in.

**Comment by tim-fitzgerald** [comment]
@alepauly ah nice one - thank you. That hadnt been updated when this issue was created. Will track there.

**Comment by alepauly** [comment]
> That hadnt been updated when this issue was created. Will track there.

Yeah, apologies for the delay updating there.

**Comment by nullcipher** [comment]
Thanks.

**Comment by nullcipher** [comment]
Hello @alepauly 

Still failing sometimes:

##[error]An error occurred while provisioning resources (Error Type: Failure).
,##[error]The remote provider was unable to process the request.
Pool: Azure Pipelines
Image: macOS-latest

Job preparation parameters
ContinueOnError: False
TimeoutInMinutes: 60
CancelTimeoutInMinutes: 5
Expand:
  MaxConcurrency: 0
  ########## System Pipeline Decorator(s) ##########

  Begin evaluating template 'system-pre-steps.yml'
Evaluating: eq('true', variables['system.debugContext'])
Expanded: eq('true', Null)
Result: False
Evaluating: resources['repositories']['self']
Expanded: Object
Result: True
Evaluating: not(containsValue(job['steps']['*']['task']['id'], '6d15af64-176c-496d-b583-fd2ae21d4df4'))
Expanded: not(containsValue(Object, '6d15af64-176c-496d-b583-fd2ae21d4df4'))
Result: True
Evaluating: resources['repositories']['self']['checkoutOptions']
Result: Object
Finished evaluating template 'system-pre-steps.yml'
********************************************************************************
Template and static variable resolution complete. Final runtime YAML document:
steps:
- task: 6d15af64-176c-496d-b583-fd2ae21d4df4@1
  inputs:
    repository: self

  MaxConcurrency: 0

Regards,

Jorge

**Comment by tim-fitzgerald** [comment]
@alepauly  This is happening again for us this morning. No indication on the status page

**Comment by tim-fitzgerald** [comment]
Again happening this morning.

**Comment by tim-fitzgerald** [comment]
And once again this morning.

**Comment by alepauly** [comment]
@tim-fitzgerald @nullcipher - yes, apologies. I believe you might have noticed before status was posted on the page. We continue working to alleviate capacity constraints. Thanks for reporting.

**Comment by mrts** [comment]
@alepauly - bringing this up again just in case you haven't noticed. The problem started again today and I see that the [status page](https://status.dev.azure.com/_event/233282345) has the following notice:

_Customers will continue to see daily queueing during peak hours for macOS pools on Azure Pipelines due to a capacity shortage. We’re adding additional capacity over the next 3 months. Until the shortage is resolved, customers should expect queueing during peak hours._

Does this apply to GitHub Actions and is it now better known when the capacity will be added?
